mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -283,6 +283,7 @@ def _compute_moc_climo_postprocess(config, runStreams, variableMap, calendar,
                        dictClimo['endYearClimo'])
     if not os.path.exists(outputFileClimo):
         print '   Load data...'
+        chunking = config.getExpression(sectionName, 'maxChunkSize')
```

Comment:
The syntax here and elsewhere makes me think that `maxChunkSize` should *always* be defined in the config file, so it should be `None` by default.
If this is the case, we should *not* comment out maxChunkSize in config.default, but rather have `maxChunkSize=None`. Is that right?